test/integration/requests/utility.ts

```diff
@@ -0,0 +1,53 @@
+import { assert } from "chai";
+import _ from "lodash";
+
+import { PingResponse, RandomResponse } from "xrpl-local";
+
+import {
+  serverUrl,
+  setupClient,
+  suiteTestSetup,
+  teardownClient,
+} from "../setupTests";
+
+// how long before each test case times out
+const TIMEOUT = 20000;
+
+describe("Utility method integration tests", function () {
+  this.timeout(TIMEOUT);
+
+  before(suiteTestSetup);
+  beforeEach(_.partial(setupClient, serverUrl));
+  afterEach(teardownClient);
+
+  it("ping", async function () {
+    const response = (await this.client.request({
+      command: "ping",
+    })) as PingResponse;
```

Comment:
shouldn't `response` be a `PingResponse` since you called `request` with `command: ping`

Reply:
`this.client` is technically type `any` because TS is confused with the this. I typed it better.